resource-aggregate/.plantuml

```diff
@@ -296,4 +296,28 @@ ResourceAggregate --> Bus : DeviceMetadataUpdated (Canceled=true)
 ResourceAggregate -> Client : CancelPendingMetadataUpdatesResponse
 deactivate Client
 deactivate ResourceAggregate
-@enduml
\ No newline at end of file
+@enduml
+
+@startuml
+title "Delete Devices"
+
+entity "OCF Server" as Server
+participant "Gateway" as Gateway
+participant "Resource Aggregate" as ResourceAggregate
+participant "Authorization Server" as AuthServer
+
+activate ResourceAggregate
+activate Server
+activate Gateway
+activate AuthServer
+Server -> Gateway : DeleteDevicesRequest
+Gateway-> ResourceAggregate : DeleteDevicesRequest
+ResourceAggregate -> Gateway : DeleteDevicesResponse
+Gateway -> AuthServer : DeleteDevicesRequest
+AuthServer -> Gateway : DeleteDevicesResponse
+Gateway -> Server : DeleteDevicesResponse
```

Comment:
Could you please include this diagram also in the plgd.dev docs, `archtiecture/component-overview/CoAP Gateway` with header Device removed by the user from the plgd Cloud ?

And make it more detailed, including interaction with the device after delete:
example:
```
...
CGW -> Server: Disconnect
destroy Server
deactivate CGW
Server -> CGW: SignUp
activate Server
activate CGW
CGW -> AS: ??? question on chat
...
CGW -> Server: Unauthorized
CGW -> Server: Disconnect
destroy Server
deactivate CGW
Server -> Server: Remove cloud credentials
```

Reply:
This MR can be closed ofc, this comment is not blocking this one :)